doc/mgmt/Show_Running_Configuration_HLD.md

```diff
@@ -0,0 +1,654 @@
+                    If exact value of parameter is to be matched then 'param=value'.
+                    
+  Example           dbpath="sonic-bgp-global:sonic-bgp-global/BGP_GLOBALS/BGP_GLOBALS_LIST/gr_restart/time
+                    dbpath="sonic-bgp-global:sonic-bgp-global/BGP_GLOBALS/BGP_GLOBALS_LIST/always_compare_med=true
+                    
+  Attribute to XML  COMMAND, PARAM
+  element
+  
+  Explanation       This tag is to map the command or param to the corresponding attribute in DB.
+                    If the CLI rendering requires some transformation, or if it uses a PTYPE then a Python callback will be needed for the command. In this case, dbpath is not relevant.
+
+
+4.  Attribute       command_tables
+
+  Syntax            Multiple sonic YANG tables separate by ";".
+                    Tables have to specify all its keys.
+                     
+  Example           command_tables="sonic-bgp-peergroup:sonic-bgp-peergroup/BGP_GLOBALS_LISTEN_PREFIX/BGP_GLOBALS_LISTEN_PREFIX_LIST/vrf_name={vrf-name},ip_prefix={ip_prfx}"
+  
+  Attribute to XML  COMMAND
+  element
+  
+  Explanation       This tag specifies the primary table for the command if it is different then view table. The keys are used from tag 'view_keys'.
+                    If attributes from more than one table are referenced in the COMMAND (other than the primary table of the view),
+                    then this command has to be rendered using command_render_cb.
+                    
+5. Attribute        command_keys
+   
+  Syntax            Multiple sonic YANG table keys separated by ','.
+                    This is optional, only required if a specific record is to be retrieved from command tables.    
+                    
+  Example           
+  
+ Attribute to XML   COMMAND
+ element
+  
+ Explanation        Specifies the keys for the command tables. The keys are required if the specific record is to be accessed for the command.  
+ 
+ Example            command_keys="instance=GLOBAL"
+
+6.  Attribute       render_command_cb
+                    
+  Syntax            String
+                      
+  Example           render_command_cb = 'bgp_confederation '
+                      
+  Attribute to XML  COMMAND
+  element
+                      
+  Explanation       In case where the CLI COMMAND or PARAM does not have direct mapping to a table attribute in the DB, the CLI developer has the option to form the
+                    command in a Python callback. The developer has to map this tag value to a callback name.
+                    The mapping is to be provided in file CLI/acioner/show_config_data.py
+                    In most cases, the command rendering is short and can be done efficiently in Python. 
+                    A Jinja template has overhead of loading which increases linearly with file size.
+                    For command rendering it is recommended to implement it in Python. 
```

Comment:
Please mention that multiple command lines should be joined using semicolon delimiter.